lib/datadog/statsd.rb

```diff
@@ -108,8 +108,10 @@ def initialize(host = DEFAULT_HOST, port = DEFAULT_PORT, opts = EMPTY_OPTIONS)
       raise ArgumentError, 'tags must be a Array<String>' unless tags.nil? or tags.is_a? Array
       @tags = (tags || []).compact.map! {|tag| escape_tag_content(tag)}
 
-      @buffer = Array.new
-      @max_buffer_size = opts[:max_buffer_size] || 50
+      # batching
+      @max_buffer_bytes = opts[:max_buffer_bytes] || 8192
```

Comment:
What are your thoughts on warning somebody when they set `max_buffer_bytes` greater than this default?